ai-ml/trainium-inferentia/addons.tf

```diff
@@ -298,13 +298,13 @@ module "eks_data_addons" {
             Name: ${module.eks.cluster_name}-node
       nodePool:
         labels:
-          - provisioner: inferentia-inf2
```

Comment:
check if this label i used by Llama2 inference ray example

Reply:
I have changed the `ray-service-llama2.yaml` to use same labels for llama2 and stable-diffusion service manifests.